test/blackbox-tests/test-cases/pkg/opam-var/opam-var-unsupported.t

```diff
@@ -0,0 +1,30 @@
+  $ . ./opam-var-helpers.sh
+
+opam variables that are explicitly unsupported in dune
+
+These should all have nice error messages explaining that they are not supported.
+
+# opam-version
+  $ test_lockfile opam-version
+  (build
+   (run echo %{pkg-self:opam-version}))
```

Comment:
Do we want to explicitly not support this or give it a dummy value? grepping the opam repo doesn't appear to show it actually being used other than being set at the top of the opam file (which we don't care about).

Reply:
If nobody is using it, we don't need to support it. I don't see of a reasonable way of using this variable.